### ISSUE TITLE
Can't include characters like Σ in SVG text element

### ISSUE THREAD

**Issue by toBeOfUse** [cduck/drawSvg#53]
When I try to save a SVG with the character Σ (sigma) in a text element, I get the error: 
```
Traceback (most recent call last):
  File "C:\Users\Mitch\Code\mc log analysis\diagrammer.py", line 401, in <module>
    drawing.saveSvg("output.svg")
  File "C:\Users\Extreme PC\AppData\Local\Programs\Python\Python39\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u03a3' in position 7970: character maps to <undefined>
```
When I try to use the XML entity \&#931; to replace it, it gets escaped and is output literally. My current solution is to use the entity and then unescape the resulting output, like:
```python
with open("output.svg", "w+") as output_file:
  output_file.write(xml.sax.saxutils.unescape(drawing.asSvg()))
```
which is a little inelegant. I think that the user should either  be able to specify whether a string that they enter for the text of a text element should be XML escaped or not or be able to explicitly specify the encoding that they want to use for the file they are outputting. (I'm using Python 3.9.1 on Windows 10; on this platform, UTF-8 encoding is (apparently) not the default.)

**Comment by cduck** [comment]
Related to #43.

**Comment by cduck** [comment]
I believe this is fixed by #65. Please reopen this issue if you see this error in drawSvg version 1.8.3 or later.
